README_SYSTEMD_GUIDANCE.md

```diff
@@ -0,0 +1,21 @@
+We have identified a service reliability issue affecting new installations of AWS IoT Greengrass Nucleus Java v2.14.0. This issue may impact service stability during system restarts and only affects fresh installations on new Greengrass devices. 
+
+Please Note:
+  - This issue does NOT affect existing Greengrass installations that have upgraded to v2.14.0 
+  - Previous versions of Greengrass Nucleus are NOT affected
+  - Only Linux devices using the default systemd service file are affected
```

Comment:
Only Linux devices using the default systemd service file **from v2.14.0** are affectetd